.cspell/computer-terms.txt

```diff
@@ -0,0 +1,354 @@
+Standardise
+xginn
```

Comment:
those are usernames and not computer terms. Can we add comments in custom dictionaries to set what each custom dictionary is for?

Reply:
While I agree with better naming, I am not very sure about maintaining a dictionary of all balena-users. Can you point to example docs where cspell is finding usernames and names of people.